lib/cog/chat/slack/template_processor.ex

```diff
@@ -90,6 +90,12 @@ defmodule Cog.Chat.Slack.TemplateProcessor do
   end
   defp process_directive(%{"name" => "list_item", "children" => children}, bullet: bullet),
     do: "   #{bullet} #{Enum.map_join(children, &process_directive/1)}\n"
+  # This is technially the default behavior in Slack. If you try to create a link with blank
+  # text only the link is displayed and is clickable, but it is included here for clarity
+  # and to conform more closely to the hipchat implementation.
+  defp process_directive(%{"name" => "link", "text" => text, "url" => url}, _) when text in [nil, ""] do
+    "<#{url}|#{url}>"
+  end
```

Comment:
If this is Slack's default behavior, you don't need  to re-implement the default just so it matches the HipChat directive processor; a comment would suffice.